### PR TITLE
Improvements added in kubectl create deployment help 

### PR DESCRIPTION
**Pull request by Ritikaa96**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it:
- Added improvement details in port, image section of `kubectl create deployment --help`
- Added example of multiple images in imperative command
#### Which issue(s) this PR fixes:
Fixes: #https://github.com/kubernetes/kubectl/issues/1457

### CODE REVIEW

**Comment by Ritikaa96** [comment]
Hi , Any kind of suggestion to improve this change is welcome. I am wondering if more detail is needed in `--field-manager`

**Comment by Ritikaa96** [comment]
All the tests are successful now, Will wait for further reviews from 
@seans3 @brianpursley

**Comment by Ritikaa96** [comment]
I'll love to know if there are more suggestions for this PR..PTAL

**Review by satyampsoni — APPROVED** [review]

**Comment by Ritikaa96** [comment]
I see , it should be in v1.29 milestone. so will wait till that.
Thanks for the approval @satyampsoni

File: staging/src/k8s.io/kubectl/pkg/cmd/create/create_deployment.go
Comment (ardaguclu) [inline staging/src/k8s.io/kubectl/pkg/cmd/create/create_deployment.go:59]: In this example, port number 5701 is only tighten to container whose image is busybox and not to others. To prevent possible misunderstandings, maybe we can remove port from here or mention this behavior. What do you think?.

Comment (ardaguclu) [reply staging/src/k8s.io/kubectl/pkg/cmd/create/create_deployment.go:59]:
```suggestion
	kubectl create deployment my-dep --image=busybox:latest --image=ubuntu:latest --image=nginx`))
```

File: staging/src/k8s.io/kubectl/pkg/cmd/create/create_deployment.go
Comment (ardaguclu) [inline staging/src/k8s.io/kubectl/pkg/cmd/create/create_deployment.go:118]: What do you mean by `It is to be noted that command can't be mentioned along with list of images.`?

**Review by ardaguclu — COMMENTED** [review]
Thanks for this PR. Left some comments, other than that it looks good to me.

File: staging/src/k8s.io/kubectl/pkg/cmd/create/create_deployment.go
Comment (Ritikaa96) [reply staging/src/k8s.io/kubectl/pkg/cmd/create/create_deployment.go:59]: > In this example, port number 5701 is only tied to container whose image is busybox and not to others. To prevent possible misunderstandings, maybe we can remove port from here or mention this behavior. What do you think?.

Hi @ardaguclu I am okay to do both , however, I'll prefer to remove the port so as it doesn't create confusion

**Comment by Ritikaa96** [comment]
> Thanks for this PR. Left some comments, other than that it looks good to me.

Thank for the suggestions , I'll make the changes.

File: staging/src/k8s.io/kubectl/pkg/cmd/create/create_deployment.go
Comment (Ritikaa96) [reply staging/src/k8s.io/kubectl/pkg/cmd/create/create_deployment.go:118]: It's been a while since I made this commit but what i can recall is that you can't write multiple container name etc.

File: staging/src/k8s.io/kubectl/pkg/cmd/create/create_deployment.go
Comment (Ritikaa96) [reply staging/src/k8s.io/kubectl/pkg/cmd/create/create_deployment.go:59]: Since i have added the commit here , i realise the command for multiple containers can also be written as 
`kubectl create deployment my-dep --image=busybox,nginx` 

should i add this example too?

File: staging/src/k8s.io/kubectl/pkg/cmd/create/create_deployment.go
Comment (Ritikaa96) [reply staging/src/k8s.io/kubectl/pkg/cmd/create/create_deployment.go:118]: i guess this makes it more confusing . so I'll remove the line.

**Comment by Ritikaa96** [comment]
Hi @ardaguclu I've made the changes .
PTAL

**Pull request merged**
